<?xml version="1.0"?>
<launch>

  <!-- send parameters to parameter server -->
  <rosparam command="load" ns="record_dataset/record_dataset" file="$(find ipa_dirt_detection)/ros/launch/record_dataset.yaml"/>


  <!--  -->
  <node pkg="ipa_dirt_detection" ns="record_dataset/record_dataset" type="record_dataset" name="record_dataset" output="screen">	<!-- ns=namespace (arbitrary), type=name of executable, name=node name (arbitrary) -->
	<!--launch-prefix="/usr/bin/gdb"-->
	<!--remap from="image_color" to="/camera/rgb/image"/-->
	<remap from="colored_point_cloud" to="/camera/depth_registered/points"/>
	<remap from="image_color" to="/camera/rgb/image_raw"/>
	<!--remap from="colored_point_cloud" to="/cam3d/depth_registered/points"/-->
	<!--remap from="colored_point_cloud" to="/cam3d/rgb/points"/-->
	<!--remap from="colored_point_cloud" to="/camera/rgb/points"/-->
	<!--remap from="colored_point_cloud" to="/cam3d/rgb/points"/-->
	
	<!-- if not an empty string, this enables using a mask that defines areas in the map where dirt detections are valid (i.e. this mask can be used to exclude areas from dirt mapping, white=detection area, black=do not detect) -->
	<!-- param name="dirtMappingMaskFilename" value="$(find ipa_dirt_detection)/common/files/map_masks/" /-->
	
  </node>

</launch>
